### ISSUE TITLE
Script doesn't fully work

### ISSUE THREAD

**Issue by Srandista** [gennariarmando/iv-centered-vehicle-cam#2]
I wanted to try this script, but I found out, that's not working as it supposed to. First of all, I tried it both with just ASI loader, and also with Fusion Fix, but the result is still the same. The part of the script, which should center the camera behind the car doesn't work, the camera is just as sluggish as it is in game by default. Script is loaded however, because when I enable the shaky camera option from VCS, the camera shakes at high speeds.

Just to be sure, the behavior shown in [this video](https://www.youtube.com/watch?v=AULYHdXmZYw) should be default one, without adjusting any settings in ini file, right?

I'm using the latest DE version, tried it also with completely clean install.

**Comment by AliakseiVenski** [comment]
Same for me. Clean steam CE install. W\o Fusion fix. Properly added ASI loader, patched scripthook. This ASI loaded for sure, camera shakes on high speed and actually controls from ini file are changing camera offsets like right, up, etc. But it is not centered at all - still th same as in original game.
@gennariarmando please help! Or at least provide more instructions on how to install this.

**Comment by gennariarmando** [comment]
Camera should look exactly the same as the video, I will  look into it soon though.

**Comment by Srandista** [comment]
Any news?

**Comment by gennariarmando** [comment]
Tested lately on latest steam release with fusion fix, works as supposed to.

**Comment by Srandista** [comment]
Ok, I saw some updates lately on GTA IV, let me check the script this evening.

**Comment by Srandista** [comment]
Unfortunately, it's still the same for me. I installed the latest GTA IV Complete update, latest Fusion Fix, and overwrite the script, just to be sure. The camera shakes during the high speeds, so the script hes been loaded, but the camera is just as sluggish like in unmodded game.

**Comment by gennariarmando** [comment]
Send screenshot please just to be sure, also you putting it into the "plugins" or "scripts" folder?

**Comment by Srandista** [comment]
Can you specify the screenshot request? Just in-game screenshot, or screenshot of same menu?

I put script to the plugins folder, next to Fusion Fix.

**Comment by gennariarmando** [comment]
A screenshot in game, while driving a car

**Comment by Srandista** [comment]
Sure, here you go.

![Image](https://github.com/user-attachments/assets/47d54ca6-2849-4b36-97ca-c7480960c0bd)

**Comment by gennariarmando** [comment]
Camera looks centered to me as it's supposed to

**Comment by Srandista** [comment]
Yeah, because I was going in straight line. That was the reason, why I asked you, in which situation I should make the screenshot. I will try to make a screen recording to show you, how it looks like on my end.

**Comment by Srandista** [comment]
Here, I don't think, that the script is centering the view as it supposed to, looks pretty much like in the unmodified game.

https://youtu.be/wjMu9-33BYI

**Comment by gennariarmando** [comment]
Yeah it's centered, this is what's supposed to do, center the camera view.

**Comment by AliakseiVenski** [comment]
But this is a behaviour from original game. It does not change anything compared to vanilla. I can't get what is the point of mod then?

**Comment by gennariarmando** [comment]
The original game has camera shifted to the left, this puts the camera to the center like in other gta's.

Before:
![Image](https://github.com/user-attachments/assets/1ed71f30-4a7a-42a9-98f1-238dfe18d9fe)

After:
![Image](https://github.com/user-attachments/assets/42ea375f-aeb8-4717-9de5-a9f0f3ff1285)

**Comment by AliakseiVenski** [comment]
Okay, I get it now. The intention of this mod is just different. Thanks for clarification.

**Comment by Srandista** [comment]
I thought (and your video suggested that as well), that one of the function of you mod is, that it'll recenter camera after the turn (to make it more similar to 3D era GTA games). Can you please clarify, if this is the case or not?

**Comment by gennariarmando** [comment]
> I thought (and your video suggested that as well), that one of the function of you mod is, that it'll recenter camera after the turn (to make it more similar to 3D era GTA games). Can you please clarify, if this is the case or not?

As the readme says:

> This plugin removes the off center vehicle camera in GTA IV

This is all this plugin does.
